app/controllers/PrepareUploadController.scala

```diff
@@ -43,7 +43,9 @@ class PrepareUploadController @Inject()(prepareUploadService: PrepareUploadServi
   def prepareUpload(): Action[JsValue] = onlyAllowedServices {
     Action.async(parse.json) { implicit request =>
       withJsonBody[UploadSettings] { (fileUploadDetails: UploadSettings) =>
-        val result: PreparedUpload = prepareUploadService.setupUpload(fileUploadDetails)
+        Logger.debug(s"Processing request: [$fileUploadDetails].")
+        val result: PreparedUpload = prepareUploadService.prepareUpload(fileUploadDetails)
+        Logger.debug(s"Returning upload template and upscan-reference: [${result}].")
```

Comment:
I would not log this as it exposes pre-signed post form, so if someone hijacks this, he can upload the file on behalf of user

Reply:
Maybe here we just can display reference?